sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionBasedLoadBalancer.java

```diff
@@ -305,7 +309,7 @@ private void closeClient() {
     private void renewOwnership(Map<String, PartitionOwnership> partitionOwnershipMap) {
         morePartitionsToClaim.set(false);
         // renew ownership of already owned partitions
-        checkpointStore.claimOwnership(partitionPumpManager.getPartitionPumps().keySet()
+        subscriptions.add(checkpointStore.claimOwnership(partitionPumpManager.getPartitionPumps().keySet()
```

Comment:
Do you know if these subscriptions get removed/cleaned up? This renewOwnership gets invoked every time there is a load balance cycle.

Reply:
I see,  so I guess it needs more investigation, I reverted this change and created this issue instead https://github.com/Azure/azure-sdk-for-java/issues/33858